src/main/scala/edu/ie3/util/scala/quantities/ApparentPower.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.util.scala.quantities
+
+import squants._
+import squants.energy._
+import squants.time.Time
+
+import scala.math.{acos, sin}
+import scala.util.Try
+
+/** Class that represents an apparent power. An apparent power is the absolute
+  * value of the complex power (|S|).
+  * @param value
+  *   power value
+  * @param unit
+  *   of the power
+  */
+final class ApparentPower private (
+    val value: Double,
+    val unit: ApparentPowerUnit,
+) extends Quantity[ApparentPower] {
+
+  def dimension: ApparentPower.type = ApparentPower
+
+  def toMillivoltamperes: Double = to(Millivoltamperes)
+  def toVoltamperes: Double = to(Voltamperes)
+  def toKilovoltamperes: Double = to(Kilovoltamperes)
+  def toMegavoltamperes: Double = to(Megavoltamperes)
+  def toGigavoltamperes: Double = to(Gigavoltamperes)
+
+  /** Method to convert this apparent power into a [[Power]] using a given
+    * cosPhi.
+    * @param cosPhi
+    *   cosine of the corresponding angle
+    * @return
+    *   the resulting active power
+    */
+  def toPower(cosPhi: Double): Power = Watts(toVoltamperes * cosPhi)
```

Comment:
Should we name this `toActivePower`? Apparent power is also one kind of power, this seems confusing to me.

Reply:
I named it this way, because `squants` uses the term `power` for its active power. But I will rename this method.